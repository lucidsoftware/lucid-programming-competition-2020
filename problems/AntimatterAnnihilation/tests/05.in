at_the_enantimatterd_there_was_nothiantimatterng_but_antiantimattermatter