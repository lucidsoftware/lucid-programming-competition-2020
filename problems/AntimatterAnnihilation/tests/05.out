at_the_end_there_was_nothing_but_